tests/codegen/common.h

```diff
@@ -18,17 +18,151 @@ namespace bpftrace {
 namespace test {
 namespace codegen {
 
-const std::string header = R"HEAD(; ModuleID = 'bpftrace'
-source_filename = "bpftrace"
-target datalayout = "e-m:e-p:64:64-i64:64-n32:64-S128"
-target triple = "bpf-pc-linux"
+#define NAME (::testing::UnitTest::GetInstance()->current_test_info()->name())
+
+static std::string rewrite_memset_call(const std::string &line)
+{
+#if LLVM_VERSION_MAJOR < 7
+  // clang-format off
+  //FROM: call void @llvm.memset.p0i8.i64(i8* nonnull align 8 %2, i8 0, i64 16,        i1 false)
+  //TO:   call void @llvm.memset.p0i8.i64(i8* nonnull         %2, i8 0, i64 16, i32 8, i1 false)
+  // clang-format on
+
+  auto parts = split_string(line, ' ');
+  std::stringstream buf;
+
+  std::string alignment;
+  for (std::vector<std::string>::iterator it = parts.begin(); it != parts.end();
+       it++)
+  {
+    if (*it == "align")
+    {
+      alignment = *(it + 1);
+      parts.erase(it, it + 2);
+      break;
+    }
+  }
+
+  parts.insert(parts.end() - 2, std::string("i32 ") + alignment + ",");
+
+  buf << parts.at(0);
+  for (unsigned int i = 1; i < parts.size(); i++)
+  {
+    buf << " " << parts.at(i);
+  }
+
+  return buf.str();
+#else
+  return std::string(line);
+#endif
+}
+
+#if LLVM_VERSION_MAJOR < 7
+static std::string rewrite_memset_decl(const std::string &line
+                                       __attribute__((unused)))
+{
+  return std::string("declare void @llvm.memset.p0i8.i64(i8* nocapture "
+                     "writeonly, i8, i64, i32, i1) #1");
+#else
+static std::string rewrite_memset_decl(const std::string &line)
+{
+  return std::string(line);
+#endif
+}
+
+static std::string rewrite_memcpy_call(const std::string &line)
+{
+#if LLVM_VERSION_MAJOR < 7
+  // clang-format off
+  //FROM: call void @llvm.memcpy.p0i8.p0i8.i64(i8* nonnull %4, i8* nonnull align 1 i32 1, %lookup_elem, i64 32, i1 false)
+  //TO:  call void @llvm.memcpy.p0i8.p0i8.i64(i8* nonnull %4, i8* nonnull %lookup_elem, i64 32, i32 1, i1 false)
+  // clang-format on
+  auto parts = split_string(line, ' ');
+  std::stringstream buf;
+  std::string alignment = "";
```

Comment:
Default initializes to empty
```suggestion
  std::string alignment;
```